ArchiSteamFarm/WebBrowser.cs

```diff
@@ -749,7 +811,15 @@ public sealed class HtmlDocumentResponse : BasicResponse {
 			[PublicAPI]
 			public readonly IDocument Content;
 
-			private HtmlDocumentResponse(BasicResponse streamResponse, IDocument document) : base(streamResponse) => Content = document;
+			internal HtmlDocumentResponse([NotNull] BasicResponse basicResponse) : base(basicResponse) { }
+
+			private HtmlDocumentResponse([NotNull] StreamResponse streamResponse, IDocument document) : base(streamResponse) {
```

Comment:
If code is calling this constructor, you should also throw if `document` is null. I don't believe there is any valid use case for this to be `null`.